### DISCUSSION TITLE
i cant install the discord.py

### DISCUSSION THREAD

**Discussion by kirigami318** [Rapptz/discord.py#6011] [Q&A] [answered]
![Screenshot_19](https://user-images.githubusercontent.com/73990179/98220913-92daff00-1f81-11eb-8fbf-30ecc5390d79.png)
like that
how to fix that?

**Comment by ShineyDev** [comment]
The [PyNaCl](https://pypi.org/project/PyNaCl/#files) library does not yet have wheels for Python 3.9. Since you're on Windows, you'd need to downgrade to Python 3.8 for now.

For future reference, the final line of the above error is asking you to install [C++ build tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/). You can read more about this process [here](https://wiki.python.org/moin/WindowsCompilers).

**Reply by kirigami318** [reply]
> The [PyNaCl](https://pypi.org/project/PyNaCl/#files) library does not yet have wheels for Python 3.9. Since you're on Windows, you'd need to downgrade to Python 3.8 for now.
> 
> For future reference, the final line of the above error is asking you to install [C++ build tools](https://visualstudio.microsoft.com/visual-cpp-build-tools/). You can read more about this process [here](https://wiki.python.org/moin/WindowsCompilers).

should i use python 3.6.6?

**Reply by ShineyDev** [reply]
Sure. Python>=3.5.3,<3.9.0 should be fine.

**Reply by Harmon758** [reply]
This is actually due to aiohttp dependencies, multidict versions < 5 and yarl versions < 1.6.1, not having wheels for Python 3.9.

**Comment by Joey451-OG** [comment]
Thanks you guys I was having the same problem

**Comment by ghost** [comment]
@Joey451-OG, you can use Python 3.8 too

**Comment by Harmon758** [comment, marked as answer]
This should be resolved with https://github.com/Rapptz/discord.py/commit/7f3737fcc2fc32078b0904131adcaa75d5eda942 in the latest development version on the master branch. See #5984.
